apps/etherpad/templates/services/app/dc.yml.j2

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: etherpad
+    service: etherpad
+    version: "{{ etherpad_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: "etherpad-app-{{ deployment_stamp }}"
+  namespace: "{{ project_name }}"
+spec:
+  replicas: {{ etherpad_app_replicas }}
+  template:
+    metadata:
+      labels:
+        app: etherpad
+        service: etherpad
+        version: "{{ etherpad_image_tag }}"
+        deploymentconfig: "etherpad-app-{{ deployment_stamp }}"
+        deployment_stamp: "{{ deployment_stamp }}"
+    spec:
+      # Prefer running pods on different nodes for redundancy
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: deploymentconfig
+                      operator: In
+                      values:
+                        - "etherpad-app-{{ deployment_stamp }}"
+                topologyKey: kubernetes.io/hostname
+      containers:
+        - name: etherpad
+          image: "{{ etherpad_image_name }}:{{ etherpad_image_tag }}"
+          imagePullPolicy: Always
+          command:
+            - "bash"
+            - "-c"
+            - node node_modules/ep_etherpad-lite/node/server.js
+                --settings {{ etherpad_configs_directory_path }}/settings.json
+                --sessionkey {{ etherpad_private_directory_path }}/SESSIONKEY.txt
+                --apikey {{ etherpad_private_directory_path }}/APIKEY.txt
```

Comment:
You should add double quote around these parameters to avoid problems with directories containing spaces.

Reply:
Please, don't tell me people are doing this?

![](https://media0.giphy.com/media/3o7TKqnN349PBUtGFO/giphy.gif)

Reply:
It's not forbidden !

![](https://media.giphy.com/media/JfJ1SrEkAyWg8/giphy.gif)

Reply:
Done.